helm_deploy/values-dev.yaml

```diff
@@ -37,9 +37,10 @@ env:
   OFFENDER_RETENTION_SQL_DATA_DUPLICATE_CHECK_ENABLED: false
   OFFENDER_RETENTION_AP_DATA_DUPLICATE_CHECK_ENABLED: false
 
-  OFFENDER_DELETION_CRON: "0 0 12-17 22 5 ? *"
+  OFFENDER_DELETION_CRON: "0 30 9 ? * MON-FRI *"
```

Comment:
Making use of the limit to keep running a daily test of the deletion in T3 (but only deleting 1 offender to avoid eating through too much test data)